latest-langs

```diff
@@ -62,6 +63,7 @@ for %langs.sort: *.key.fc -> (:key($name), :value(%lang)) {
         state $ua  = HTTP::Tiny.new :throw-exceptions;
         my    $res = $ua.get("https://$_")<content>.decode;
 
+        when / 'aplwiki'   / { $res ~~ / ' release' .+? <(\d+ <[\d.]>+)> / }
```

Comment:
I'm fully aware of that line, and how Dyalog APL's the only dialect of a language that'll make use of the domain unless we add additional dialects. Regardless, it serves its purpose well.